lib/LedgerNodeOperations.js

```diff
@@ -25,7 +25,7 @@ module.exports = class LedgerNodeOperations {
    *
    * @return {Promise} resolves once the operation completes.
    */
-  async add({operation}) {
+  async add({operation, options}) {
```

Comment:
Would be good to fix `@param operation` => `@param options.operation` and add `@param options.options - Consensus plugin-specific options to pass through.` (or similar).

Reply:
Logged as https://github.com/digitalbazaar/bedrock-ledger-node/issues/95 .. I'll see how easy/hard it is to do in the `v1-dingo-rc1` branch.

Reply:
That's fine, but note that this wouldn't be breaking anything, it's just a documentation change. The docs are broken for `operation` and missing for `options`.